src/action.c

```diff
@@ -357,14 +359,16 @@ action_arg_from_xml_node(struct action *action, const char *nodename, const char
 		}
 		/* Falls through to GoToDesktop */
 	case ACTION_TYPE_GO_TO_DESKTOP:
-		if (!strcmp(argument, "to")) {
-			action_arg_add_str(action, argument, content);
-			goto cleanup;
-		}
 		if (!strcmp(argument, "wrap")) {
 			action_arg_add_bool(action, argument, parse_bool(content, true));
 			goto cleanup;
 		}
+		/* Falls through to MoveToOutput */
+	case ACTION_TYPE_MOVE_TO_OUTPUT:
+		if (!strcmp(argument, "to")) {
```

Comment:
I think "direction" is more in line with other actions that manipulate window relocation. The "to" value is only used for desktop operations and also includes "last" and "current" which run afoul of the directional concept applied here.

If this change is adopted, I recommend changing relevant variables and function arguments to remain consistent.